docs/setup/install-with-keadm.md

```diff
@@ -1,354 +1,261 @@
 ---
+
 title: Installing KubeEdge with Keadm
 sidebar_position: 3
 ---
 
-Keadm is used to install the cloud and edge components of KubeEdge. It does not handle the installation of Kubernetes and its [runtime environment](https://kubeedge.io/docs/setup/prerequisites/runtime).
+Keadm is used to install the cloud and edge components of KubeEdge.  
+It does **not** handle Kubernetes installation or its [runtime environment](https://kubeedge.io/docs/setup/prerequisites/runtime).
+
+Check [Kubernetes compatibility](https://github.com/kubeedge/kubeedge?tab=readme-ov-file#kubernetes-compatibility) to confirm supported versions.
```

Comment:
We have already mentioned compatibility here and provided the link, so there is no need to repeat it later from Line20. Doing so would only result in having to maintain updates for compatibility in two places.